### PR TITLE
adding example sections for DBaaS API

### PR DESCRIPTION
**Pull request by denisok**
[PMM-8222](https://jira.percona.com/browse/PMM-8222)

Co-authored-by: Nurlan Moldomurov <nurlan.moldomurov@percona.com>
Co-authored-by: Denys Kondratenko <denys.kondratenko@percona.com>

### CODE REVIEW

File: markdown/dbaas.md [hunk 1]
```diff
@@ -31,3 +31,48 @@ PMM provides set of API calls to enable DBaaS, configure it and to create and ma
...
+curl -X POST "http://localhost/v1/management/DBaaS/Kubernetes/Register" -H "accept: application/json" -u "admin:admin" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"kube_auth\": { \"kubeconfig\": \"${KUBECONFIG}\" }}"
+```
+This command will register kubernetes cluster, start monitoring of kubernetes cluster and install required kubernetes operators.
+
+API endpoint used in this step: [RegisterKubernetesCluster](https://percona-pmm.readme.io/reference/registerkubernetescluster)
+
+### Create PXC Cluster
+
+Once we registered kubernetes cluster we can use it’s name to create DB Clusters. Here is an example for PXC Cluster.
+
```
Comment (BupycHuk) [inline markdown/dbaas.md:73]: should we mention that values used here is a recommended ones?

File: markdown/dbaas.md [hunk 2]
```diff
@@ -31,3 +31,93 @@ PMM provides set of API calls to enable DBaaS, configure it and to create and ma
 - https://percona-pmm.readme.io/reference/registerkubernetescluster
 - https://percona-pmm.readme.io/reference/createpxccluster
 - https://percona-pmm.readme.io/reference/createpsmdbcluster
+
+In this example we would use minikube for the kubernetes cluster and create PXC DB cluster, but similar API endpoints exist for the PSMDB.
```
Comment (percona-csalguero) [inline markdown/dbaas.md:35]:
```suggestion
In this example we would use minikube for the kubernetes cluster and will create a PXC DB cluster, but similar API endpoints exist for the PSMDB.
```

File: markdown/dbaas.md (hunk 1 above)
Comment (percona-csalguero) [reply markdown/dbaas.md:74]: Yes

File: markdown/dbaas.md [hunk 3]
```diff
@@ -31,3 +31,93 @@ PMM provides set of API calls to enable DBaaS, configure it and to create and ma
...
+
+```bash
+curl -X POST "http://localhost/v1/management/DBaaS/PXCCluster/Create" -H "accept: application/json" -u “admin:admin” -H "Content-Type: application/json" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"name\": \"my-cluster-1\", \"expose\": true, \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } }}"
+```
+
+API endpoint used in this step: [CreatePXCCluster creates](https://percona-pmm.readme.io/reference/createpxccluster).
+
+### List Kubernetes clusters
+
+Once you created PXC cluster you can check the status of the cluster by calling `List` endpoint.
```
Comment (percona-csalguero) [inline markdown/dbaas.md:83]:
```suggestion
Once you created PXC cluster you can check the status of the cluster by calling the `List` endpoint.
```

**Review by percona-csalguero — CHANGES_REQUESTED** [review]

File: markdown/dbaas.md [hunk 4]
```diff
@@ -31,3 +31,292 @@ PMM provides set of API calls to enable DBaaS, configure it and to create and ma
...
+
+Once kubernetes cluster is created it should be registered in PMM where `my_cluster` is a name of kubernetes cluster which will be used later. `sed` command is used to remove newlines, otherwise this script doesn’t work.
+```bash
+KUBECONFIG=$(kubectl -- config view --flatten --minify | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\n/g')
+
+curl -X POST "http://localhost/v1/management/DBaaS/Kubernetes/Register" -H "accept: application/json" -u "admin:admin" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"kube_auth\": { \"kubeconfig\": \"${KUBECONFIG}\" }}"
+```
+This command will register kubernetes cluster, start monitoring of kubernetes cluster and install required kubernetes operators.
+
+API endpoint used in this step: [RegisterKubernetesCluster](https://percona-pmm.readme.io/reference/registerkubernetescluster)
```
Comment (atymchuk) [inline markdown/dbaas.md:69]:
```suggestion
API endpoint used in this step: [RegisterKubernetesCluster](ref:registerkubernetescluster)
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

File: markdown/dbaas.md [hunk 5]
```diff
@@ -27,7 +27,334 @@ To configure and use DBaaS you would need to have PMM deployment and Kubernetes
...
+            "status": "available"
+          },
+          "8.0.20-11.1": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.20-11.1",
+            "image_hash": "54b1b2f5153b78b05d651034d4603a13e685cbb9b45bfa09a39864fa3f169349",
+            "status": "available"
+          },
+```
+
+The chosen `image_path` value is the value you should provide in the next API call as the `image` field.
```
Comment (BupycHuk) [inline markdown/dbaas.md:244]:
```suggestion
The chosen `image_path` value is the value you should provide in the next API call as the `image` field. We recommend using the one with `"status": "recommended"`
```

**Review by BupycHuk — DISMISSED** [review]

**Review by BupycHuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]

**Pull request merged**